input/input_test.go

```diff
@@ -0,0 +1,376 @@
+/*
+Package Input reads config file and populates the data in Config struct.
+*/
+package input
```

Comment:
Should input_test.go be in the same folder as input.go? or all test files should go under "tests" directory?

Reply:
Hi @aswinbh as per golang , unit test files needs to be present in same folder as package.